crates/eth-rpc/src/api/kakarot_custom_api.rs

```diff
@@ -0,0 +1,14 @@
+use jsonrpsee::proc_macros::rpc;
+use kakarot_rpc_core::models::balance::TokenBalances;
+use reth_primitives::Address;
+
+#[rpc(server)]
+#[async_trait]
+pub trait KakarotCustomApi {
+    #[method(name = "kakarot_getTokenBalances")]
```

Comment:
isn't there a way (I think there is) to automatically prepend the `alchemy_` (here `kakarot_`) to all `#[methods()]` (maybe create an issue and let's update this for the eth api as well)

Reply:
Yes, we can create namespaces and do that, I am planning a next PR after it for that,

because:
- We would have to do that in EthApi as well
-  I can see net_version doesn't start with "eth"
-  which means we probably would have to refactor a new trait for it, and I think in that case I would prefer to do it in another PR, lemme know what you think. 
<img width="703" alt="image" src="https://github.com/kkrt-labs/kakarot-rpc/assets/41180869/7a74162f-7337-4a5f-a9ba-78ba8698f3a1">